recipes-wpe/wpeframework/wpeframework-plugins_git.bb

```diff
@@ -75,7 +78,7 @@ PACKAGECONFIG[webserver]      = "-DPLUGIN_WEBSERVER=ON \
                                  -DPLUGIN_WEBSERVER_PATH="${PLUGIN_WEBSERVER_PATH}" \
                                  ,-DPLUGIN_WEBSERVER=OFF,"
 PACKAGECONFIG[webshell]       = "-DPLUGIN_WEBSHELL=ON,-DPLUGIN_WEBSHELL=OFF,"
-PACKAGECONFIG[wifi]           = "-DPLUGIN_WIFICONTROL=ON,-DPLUGIN_WIFICONTROL=OFF,,wpa-supplicant"
+PACKAGECONFIG[wifi]           = "-DPLUGIN_WIFICONTROL=ON,-DPLUGIN_WIFICONTROL=OFF,,wpa-supplicant linux-firmware-bcm43430"
```

Comment:
is the linux-firmware-bcm4340 a generic package? You need it right now for the raspberry pi only. It will not work for the other platforms like nexus, as they don't provide such package. This will cause issues

Reply:
No this needs to be changed to an RPI only depends, e.g. using _rpi on a variable and then ${thatvar} here.